### ISSUE TITLE
error CS7036: There is no argument given that corresponds to the required parameter 'value'

### ISSUE THREAD

**Issue by alexeybut** [microsoft/node-api-dotnet#238]
The error occurs for the sample project:
```
Microsoft.JavaScript.NodeApi.Generator\Microsoft.JavaScript.NodeApi.Generator.ModuleGenerator\Conso leApp1.NodeApi.g.cs(31,21): error CS7036: There is no argument given that corresponds to the required parameter 'value' o f 'class2.class2(FileStream, object)' [ConsoleApp1.csproj] 
```

The reason is in the constructor:
```
internal class2(FileStream stream, object value)
{
}
```

[ConsoleApp1.zip](https://github.com/microsoft/node-api-dotnet/files/14694192/ConsoleApp1.zip)

Command:
`
dotnet publish -c DEBUG -r win-x64
`
0.5.16 nuget version
Seems related to #232

**Comment by jasongin** [comment]
Thanks for the bug report! #240 has the fix.

**Comment by jasongin** [comment]
Fixed in 0.5.17.
